PG-autoloads-bootstrap.el

```diff
@@ -0,0 +1,18 @@
+;;;###autoload
+;;; Bootstrap ProofGeneral's autoloads.
+;; The code in package.el will not look in sub-directories of "the package directory"
+;; \(c.f. `package-generate-autoloads')
+;; , so it needs a little help from the code below to find ProofGeneral's autoloads.
+(let* ((base-dir (file-name-directory
```

Comment:
It looks like this code will run every time the file is loaded.  An `eval-when-compile` wrapper would probably help here :)